src/main/java/com/hubspot/jinjava/el/ext/JinjavaListELResolver.java

```diff
@@ -30,12 +31,60 @@ public boolean isReadOnly(ELContext context, Object base, Object property) {
   @Override
   public Object getValue(ELContext context, Object base, Object property) {
     try {
+      // If we're dealing with a negative index, convert it to a positive one.
+      if (isResolvable(base)) {
+        int index = toIndex(property);
+        List<?> list = (List<?>) base;
+        if (index < 0) {
+          // Leave the range checking to the superclass.
+          property = index + list.size();
```

Comment:
`list` can be inlined.

Reply:
https://github.com/HubSpot/jinjava/pull/245/commits/6cda4efc04ad96459c8ad4287ec0c88a454cd931